Tasks/cURLUploader/curluploader.ts

```diff
@@ -41,6 +41,7 @@ async function run() {
         var curlRunner: trm.ToolRunner = tl.tool('curl');
 
         // Resolve files for the specified value or pattern
+        let uploadCount = 1;
```

Comment:
Two paths: 
1) they upload only one file (this gets that path).
2) they use wildcard:
2a) if  no files, error (which would cover your concern, correct?)
2b) if there are files, then line 76 will correct the value

Are you seeing some other flaw?